### PR TITLE
Add link to web translation tool

### PR DESCRIPTION
**Pull request by 6543**
... and a new gitignore entry

@helloworld1 I'll create pulls from time to time with updates from Crowdin if there are some
If you like admin access to the crowdin project just write to me :)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,6 +19,14 @@ Most part of the code is re-written with modern Jetpack libraries and Kotlin lan
...
 <img src="https://play.google.com/intl/en_us/badges/images/generic/en-play-badge.png" alt="Get it on Google Play" height="80"/></a>
 
-# Build FreeOTP
+# Build Dependencies
 * Android SDK
 * Android Studio 4.0+
+
+# Translate
+
+[Crowdin](https://crowdin.com/project/freeotpplus) is used for translation.
```
Comment (helloworld1) [inline README.md:28]: I will rephrase the "is used for translation" to "can be used for translation if you are uncomfortable working with strings.xml files."

**Comment by helloworld1** [comment]
Thanks for setting up Crodwin. It looks pretty nice!

**Comment by 6543** [comment]
@helloworld1 updated

**Comment by 6543** [comment]
@helloworld1 updated

**Pull request merged**